_posts/2025-03-26-kubeflow-1.10-release.md

```diff
@@ -0,0 +1,272 @@
+|       Manifests        |   14   |      8       |    4     |     94     |    52   |
+|        Trainer         |   1    |      0       |    0     |     1      |    0    |
+|     Model Registry     |   6    |      1       |    13    |    153     |   188   |
+|         Spark          |   1    |      5       |    37    |    1640    |   141   |
+|       All Images       |   81   |     115      |   1009   |   13275    |   4804  |
+
+## Pipelines
+
+Kubeflow Pipelines 2.4.1 introduces several enhancements to improve the flexibility and control of your machine learning
+workflows.
+
+### Support for Placeholders in Resource Limits
+
+Kubeflow Pipelines 2.4.1 introduces support for [placeholders in resource limits](https://github.com/kubeflow/pipelines/pull/11501),
+enhancing flexibility in pipeline execution.This update allows users to define dynamic resource limits using
+parameterized values, enabling more adaptable and reusable pipeline definitions.
+
+### Support for Loop Parallelism
+
+Kubeflow Pipelines 2.4.1 introduces a new [Parallelism Limit for `ParallelFor` tasks](https://github.com/kubeflow/pipelines/issues/8718),
+giving users the ability to run massively parallel inference pipelines, with more control over parallel execution in
+their workflows. This feature allows users to specify the maximum number of parallel iterations, preventing resource
+overutilization and improving system stability.
+
+### Implement SubDAG Output Resolution
+
+Kubeflow 1.10 ensures that [pipelines using nested DAGs](https://github.com/kubeflow/pipelines/pull/11196) work
+correctly and reliably when treated as components. Outputs from deeply nested DAGs will now resolve properly, avoiding
+broken dependencies.
+
+## Model Registry
+
+Model Registry introduces a new user interface and enhanced model management capabilities.
+
+### Model Registry UI
+
+The new Kubeflow [Model Registry UI](https://www.kubeflow.org/docs/components/model-registry/getting-started/#using-the-model-registry-ui)
+provides a user-friendly web interface for managing machine learning models within the Kubeflow platform. It centralizes
+model metadata, version tracking, and artifact management, streamlining MLOps workflows.
+
+Key features include:
+
+* Easy model registration with custom metadata
+* Comprehensive model management with filtering and sorting
+* Archiving capabilities
+* Version control
+* Metadata editing
+
+![Model Registry UI](../images/2025-03-26-kubeflow-1.10-release/model-registry-ui.png)
+
+The UI interacts with the Model Registry's REST API, making it accessible to users of all technical backgrounds and
+enhancing collaboration across data science, ML engineering, and MLOps teams.
+
+To get started with the Model Registry UI, which is currently in Alpha, you can follow the instructions
+[here](https://www.kubeflow.org/docs/components/model-registry/installation/#installing-on-kubeflow-platform).
+
```

Comment:
```suggestion

The Kubeflow Model Registry UI Team would like to conduct user research to identify possible enhancements we can contribute in future iterations of the Kubeflow Model Registry UI. If you are interested in participating in this study, please fill out [this survey](https://docs.google.com/forms/d/e/1FAIpQLSeCveL-b0NyUohYa86I3VeTXeynEQLpV5Loj-1HkoUVDwlVCQ/viewform).
```

Reply:
@dpoulopoulos @rimolive I've added here and saw it accepted. I wonder if https://github.com/kubeflow/blog/pull/166/commits/aeea0e8f6eaf266fce2dbc587efbf387c429979d was reverted by accident?